### PR TITLE
Update favicons 5.x 2022-11-30

### PR DESCRIPTION
**Pull request by dodmi**

### CODE REVIEW

File: data/favicon.json [hunk 1]
```diff
@@ -406,6 +406,7 @@
 	"tidbit5150.com": "tidbit5150.com.png",
 	"timbusinessmail.it": "timbusiness.it.ico",
 	"tk.de": "tk.de.ico",
+	"tracker-software.com": "tracker-software.com.png",
```
Comment (lieser) [inline data/favicon.json:409]: There did you get the icon from? If I visited the site today it did show a different favicon (https://trackerwebsitestatic.cachefly.net/cdn/bundles/app/images/favicon.ico)

File: data/signersDefault.json [hunk 2]
```diff
@@ -85,7 +85,7 @@
 			"sdid": "aliexpress.com mail.aliexpress.com",
 			"ruletype": "ALL",
 			"priority": "DEFAULT_RULE_ALL_2"
-		}, {
+ 		}, {
```
Comment (lieser) [inline data/signersDefault.json:88]: There seem to be a lot unintentionally whitespace changes.

**Review by lieser — CHANGES_REQUESTED** [review]

File: data/favicon.json (hunk 1 above)
Comment (dodmi) [reply data/favicon.json:409]: Actually, I had it locally stored (for use with another addon). I just recently saw, that their mails use dkim signatures, but didn't look at their homepage, which got a redesign.
https://web.archive.org/web/20140308061603/http://www.tracker-software.com/
I've updated the icon to the latest one.

File: data/signersDefault.json (hunk 2 above)
Comment (dodmi) [reply data/signersDefault.json]: Sorry. I replaced the spaces with tabs (again)

**Review by lieser — APPROVED** [review]

**Pull request merged**